src/ts/app/console/src/render.ts

```diff
@@ -0,0 +1,28 @@
+import {html, TemplateResult} from "lit-html"
+import {ConsoleFragment} from "./subtree"
+
+
+
+export function render(fragments: ConsoleFragment[]): TemplateResult {
+    return html`
+        <style>
+        
+        div {
```

Comment:
I must not be understanding how the CSS scoping is working... this looks super dangerous to me, I would expect at least a class attached to the divs... but does this get scoped to only this component somehow? Who does that? lithtml?

Reply:
Shadow dom! check out the subscribers...or just do a cross-project case-insensitive search for "shadow".